domain-xample/xample-workflows/src/main/resources/conf-camel.yml

```diff
@@ -7,3 +7,10 @@ camel.dataformat.json-jackson.auto-discover-object-mapper: true
 # Ensure the Spring-Boot application keeps running until being stopped or the JVM terminated
 # https://camel.apache.org/components/2.x/others/spring-boot.html
 camel.springboot.main-run-controller: true
+
+vro:
+  camel:
+    # Can use specific class or all classes in package name (using suffix '.*')
+    # Parsed by CamelDtoClassesProperties. `gov.va.vro.model.*` is already included
+    dto-classes:
+      - gov.va.vro.biekafka.model.*
```

Comment:
Reminder to enable [pre-commit hooks](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Development-process#pre-commit-hook) in your local env.

Reply:
Initially i started to make this implementation a little more specific, but thought that fell into the purview of #1680, so kept this iteration more general.